canonical-websites/www.ubuntu.com
# Move conjure-up to try-openstack

## Done
Move download/cloud/conjure-up to /download/cloud/try-openstack and update the content.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/cloud/try-openstack](http://0.0.0.0:8001/download/cloud/try-openstack)
- Check that the copy matches the [copy doc](https://docs.google.com/document/d/1TXnoE_RZCgms1K3beJNjzYLNzZEIjfeuNYg_zhkYQTA/edit#heading=h.wc7vrfwrcrio)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2185
